edgexfoundry/edgex-compose
# refactor!: Remove deprecated unsupported service

BREAKING CHANGE: System, Device Camera and Device Grove are no longer included or added to generated compose file. Use new Device ONVIF Camera service as alternative. Others have no replacement.

closes #317

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD


## Testing Instructions
Run `make run ds-virtual`
Verify all expected services are running and the `system` service is not running
Run `make clean`
Run `make run ds-camera`
Verify `ERROR: No such service: ds-camera` is displayed
Run `make run ds-grove`
Verify `ERROR: No such service: ds-grove` is displayed
